### PR TITLE
add howto on simple measure of single value

### PR DESCRIPTION
**Pull request by joachimR2301**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [X] Pull request title reflects its content
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Description:
I found it useful to be able to get a measurement value in a simple way and think it should be highlighted in the how_to docs

### CODE REVIEW

File: docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Sometimes you just want to get a single measurement value of a signal and do not need a streaming session.
+This can be easily done by using the getLastValue() method:
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+void HowToDoc_MeasureSingleValue(daq::DevicePtr device)
```
Comment (denise-opendaq) [inline docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc:15]: Example on Python is missing

File: docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    // get 1st signal of device...
+    auto signal = device.getSignalsRecursive()[0];
+    // ...and get a measurement value
+    float measurementValue = signal.getLastValue();
```
Comment (denise-opendaq) [inline docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc:20]: Last value can be not assigned or have another type

File: docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
+= Measure a single value
+
+[#measure_single_value]
+== Measure a single value
+
+Sometimes you just want to get a single measurement value of a signal and do not need a streaming session.
```
Comment (denise-opendaq) [inline docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc:6]: You can replace text with something like 

OpenDAQ offers a variety of solutions for reading signal values, including readers. For developers requiring access to only the most recent value, the signal interface provides a method named "getLastValue."

File: docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc (hunk 1 above)
Comment (joachimR2301) [reply docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc:14]: example added

File: docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc (hunk 2 above)
Comment (joachimR2301) [reply docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc]: fair enough, my device delivered float so it was working fine. Adapted example

File: docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc (hunk 3 above)
Comment (joachimR2301) [reply docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc]: Description updated

**Review by JakaMohorkoDS — APPROVED** [review]
Thank you for contributing to our documentation! The addition looks good to me with the recent changes.

**Pull request merged**